app/models/report.rb

```diff
@@ -85,4 +85,8 @@ def self.expire(conditions = {})
   def no_report
     false
   end
+
+  def self.origins
+    all.pluck(:origin).uniq.compact
```

Comment:
shouldn't we use registered reporters instead? this will return different origins based on what reports we have in db, reports can be cleaned or not created yet even though you expect to have e.g. puppet and ansible reports